StringCalculator.Tests/StringCalculatorTests.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using Xunit;
+
+namespace StringCalculator.Tests
+{
+    public class StringCalculatorShould
+    {
+        [Fact]
+        public void ReturnZeroFromEmptyString()
+        {
+            var calculator = new StringCalculator();
+            var expected = 0;
+
+            var actual = calculator.Add("");
+            
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ReturnNumberFromSingleStringNumber()
+        {
+            var calculator = new StringCalculator();
+            var expected1 = 1;
+            var expected2 = 3;
+
+            var actual1 = calculator.Add("1");
+            var actual2 = calculator.Add("3");
+
+            Assert.Equal(expected1, actual1);
+            Assert.Equal(expected2, actual2);
+        }
+
+        [Fact]
+        public void ReturnSumFromTwoNumber()
+        {
+            var calcualtor = new StringCalculator();
+            var expected1 = 3;
+            var expected2 = 8;
+
+            var actual1 = calcualtor.Add("1,2");
+            var actual2 = calcualtor.Add("3,5");
+
+            Assert.Equal(expected1, actual1);
+            Assert.Equal(expected2, actual2);
+        }
+
+        [Fact]
+        public void ReturnSumFromAnyAmountOfNumbers()
+        {
+            var calculator = new StringCalculator();
+            var expected1 = 6;
+            var expected2 = 20;
+
+            var actual1 = calculator.Add("1,2,3");
+            var actual2 = calculator.Add("3,5,3,9");
```

Comment:
Would be good to think of more descriptive variable names. @pnakkala is there a better way to do multiple assertions under one test name?

Reply:
I have heard somewhere people saying a unit test method should only have one assertion. In practice, i have seen multiple assertions under one test method

Reply:
That said, current name seems alright to me